Update wording (#590)
longhorn/longhorn-tests: docs/content/manual/pre-release/upgrade/auto-upgrade-engine.md

7. Take backup for a volume when upgrade hasn't started.
8. Create 10 volumes and set concurrent upgrade limit to 2. When the upgrade starts, delete all the volumes. [Volumes are deleted and Longhorn UI is not corrupted.
9. Deploy a workload with rwx scale 10 with persistent volume claim template and verify the volume created can be used for all of 10 pods.

> 10. While upgrade has started, detach the volume from node and verify the detached volume is upgraded.

Reviewer: ```suggestion
10. While upgrade is in progress, detach the volume from node and verify the detached volume is upgraded.
```
